CHANGELOG.md

```diff
@@ -2,8 +2,17 @@
 
 ### Release Notes
 
+- On systemd Telegraf will no longer redirect it's stdout to /var/log/telegraf/telegraf.log.
+On most systems, the logs will be directed to
+the systemd journal and can be accessed by `journalctl -u influxdb.service`.
```

Comment:
Fix the formatting here so the lines are wrapped at the correct portion of the file. Also, switch `influxdb` to `telegraf`.